modules/ROOT/pages/dashboards/advanced-features/filters-and-parameters.adoc

```diff
@@ -0,0 +1,87 @@
+= Filters and parameters
+:description: Use parameters in your Aura dashboards to further customize queries or use them in filters for node and relationship properties.
+
+
+== Parameters
+
+Parameters are associated with a specific dashboard.
+Each parameter has a value and type both of which you can set initially. 
+
+Display the parameters of a dashboard by using the parentheses button to show the **Parameters** drawer.
+
+.The parameters drawer
+image::dashboards/parameters-drawer.png[]
+
+The drawer shows the dashboard parameters in alphabetical order.
+Each parameter has an icon next to it hinting at its data type:
+
+* A text icon for strings,
+* A hashtag icon for numbers,
+* A calendar icon for dates,
+* A filter icon for unspecified data types. 
+
+
+=== Creating a parameter
+
+. From the **Parameters** drawer, use the **Add** button.
+. In the dialog, enter a parameter name, select a data type and optionally an initial value for the parameter, then **Save**.
+
+
+=== Editing a parameter
+
+. From the **Parameters** drawer, hover over the parameter you wish to edit.
+. Use the pen button that appears.
+. In the dialog, edit the parameter name, data type and/or initial value, then **Save**.
+
+
+=== Using a parameter in a query
+
+To use a parameter in a query, use the parameter name prefixed with `$`, for example `$example_number_param` for the parameter with name "example_number_param".
+
+.Using a parameter in a query
+image::dashboards/parameter-in-query.png[]
+
+
+=== Deleting a parameter
+
+. From the **Parameters** drawer, hover over the parameter you wish to edit.
+. Use the trash bin button that appears.
+
+
+== Filters
+
+Filters are special xref:dashboards/working-with-dashboards/managing-dashboards.adoc#_dashboard_cards[cards] and you can add them to your dashboards in a similar fashion.
+
+You can use a filter to compare a node or relationship property to a parameter value. 
+
+
+== Adding a filter
+
+. In the **Dashboards** page, use **Add filter** at the bottom right.
+. In the **Add filter** dialog, select a filter type, a node or relationship label and a node or relationship property.
```

Comment:
should be changed to refer to node labels and relationship types:
"select a filter type: a node label or relationship type, and a property."